cmd/volume.go

```diff
@@ -76,20 +99,71 @@ func volumeListCommand() *cobra.Command {
 	return cmdVolumeList
 }
 
+func volumeUpdateCommandHandler(cmd *cobra.Command, args []string) {
+	name := args[0]
+	label := args[1]
+	config, _ := cmd.Flags().GetString("config")
+	provider, _ := cmd.Flags().GetString("provider")
+	conf := unWarpConfig(config)
+
+	var vol api.VolumeService
+	var err error
+	if provider == "onprem" {
+		vol = &api.OnPrem{}
+	} else {
+		vol, err = getCloudProvider(provider)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+
+	conf.BuildDir = api.LocalVolumeDir
+
+	err = vol.UpdateVolume(conf, name, label)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func volumeUpdateCommand() *cobra.Command {
+	cmdVolumeUpdate := &cobra.Command{
+		Use:   "update <volume_name:volume_uuid> <new_label>",
```

Comment:
I don't know if there are specific requirements to allow a user to update the label of an existing volume, but this won't work: the label specified when creating a volume with mkfs cannot be changed later on (it's encoded in the .raw file). You can change the symlink for an on-prem volume file, but its internal label will remain the same, so you won't be able to use the new label in a `mounts` directive in a manifest.

Reply:
oh yeah it makes sense, will remove this